http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/util/AKSInterpreter.scala

```diff
@@ -176,18 +175,21 @@ class AKSInterpreter[F[_]](config: AKSInterpreterConfig,
         app.appType,
         params.workspaceId,
         app.appName,
-        refererConfig.validHosts + relayDomain
+        refererConfig.validHosts + relayDomain,
+        config.samConfig,
+        config.listenerImage,
+        config.leoUrlBase
       )
 
       _ <- logger.info(ctx.loggingCtx)(
-        s"Setup chart values for app ${params.appName.value} are ${values.asString}"
+        s"Relay listener values for app ${params.appName.value} are ${values.asString}"
       )
 
       _ <- helmClient
         .installChart(
-          getTerraAppSetupChartReleaseName(app.release),
```

Comment:
I think I am a bit confused now, how are we installing the rest of the setup (pod identity etc.) now?

Reply:
With the listener deployment moved, there are no longer any Azure things needed in the setup chart!

Reply:
Ohhh I missed that completely, in my mind the pod identity was deployed via this I don't know why 😓 In that case, this makes perfect sense!